scripts/scene/EditingManager.gd

```diff
@@ -33,11 +33,6 @@ func _ready() -> void:
 	# Connect the buttons to the confirmation functions in the GUI script
 	popup = GUI.manager.new_popup(room_editor.popup_message, false, room_editor.confirm_delete, room_editor.cancel_delete)
 	
-func _input(event):
-	if event is InputEventMouseButton:
-		if event.pressed:
-			match event.button_index:
-				1: room_editor.handle_select_input(event, camera.offset, camera.zoom);
 
```

Comment:
According to @jazmand, the _input is always listening for click, but the load_room_types is just called once, right?

Reply:
yeah the load_room_types is just called once on _init, it is not related to the _input function